<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-HE426NT" failures="1" tests="4" name="Default test" time="8.259" errors="0" timestamp="2021-05-26T13:25:20 IST">
  <testcase classname="com.framework.tests.ResetPassword" name="login" time="1.358"/>
  <testcase classname="com.framework.tests.ResetPassword" name="startTest" time="0.003"/>
  <testcase classname="com.framework.tests.Adminlogin" name="login" time="0.003">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.framework.utilities.Readme.waitforvisibile(String)&amp;quot; because &amp;quot;this.action&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.framework.utilities.Readme.waitforvisibile(String)" because "this.action" is null
at com.framework.tests.Adminlogin.login(Adminlogin.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="com.framework.tests.Adminlogin" name="startTest" time="0.002"/>
  <testcase classname="com.framework.tests.Baseclass" name="@AfterTest Teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="com.framework.tests.Baseclass" name="@AfterTest Teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="com.framework.tests.Baseclass" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="com.framework.tests.Baseclass" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
</testsuite> <!-- Default test -->
